pkg/operator/testlib/vcsim_framework.go

```diff
@@ -83,6 +83,9 @@ func SetupSimulator(modelDir string) (*vclib.VSphereConnection, func(), error) {
 		Username: "admin@vsphere.local",
 		Password: "foobar",
 	}
+	/*connections := []*vclib.VSphereConnection{
+		conn,
+	}*/
```

Comment:
can we drop this commented code or are we leaving it in for historical reasons?

Reply:
yes